src/wrapper.js

```diff
@@ -1,32 +1,34 @@
-import { createChart, createControls, createTable } from 'webcharts';
-import { controlInputs, syncControlInputs, syncSettings } from './default-settings'
-import config from './default-settings';
+import './util/object-assign';
+import defaultSettings from './defaultSettings';
+import { syncSettings, syncControlInputs, controlInputs } from './defaultSettings';
+
+import { createControls, createChart, createTable } from 'webcharts';
+
 import onInit from './onInit';
 import onLayout from './onLayout';
 import onDataTransform from './onDataTransform';
 import onDraw from './onDraw';
 import onResize from './onResize';
-import './util/object-assign';
 
-export default function yourFunctionNameHere(element, settings){
-	
-	//merge user's settings with defaults
-	let mergedSettings = Object.assign({}, config, settings);
+export default function safetyResultsOverTime(element, settings) {
+    
+  //Merge user settings onto default settings.
+    let mergedSettings = Object.assign({}, defaultSettings, settings);
+
```

Comment:
We should bite the bullet and update the way we merge settings eventually, but not for this round of releases ... likely to bite us at some point though ...